### ISSUE TITLE
After new install using site-regular and using custom URL for admin cannot access admin URL

### ISSUE THREAD

**Issue by blackdoginet** [YunoHost-Apps/processwire_ynh#4]
Just installed processwire in yunohost using site-regular profile. I have given my admin URL a secure name. I now cannot access the admin URL. I keep getting redirected to the yunohost web panel.
Also none of the links on the home page direct correctly again I go back to Yunohost start page

**Comment by kathep** [comment]
I am having a similar issue with the processwire package. After successful install, the link to my admin page serves a 404 nginx error.

@ericgaspar Can you suggest a change/edit necessary to make admin page connect successfully?

I love using PW and YH, and it would be wonderful to be able to successfully use them together.

**Comment by alfredo87** [comment]
Yep, this issue is very much the same: installing via web admin in yuno the app does not allow one to write new posts or run admin dashboard: useless as it stands. A pity because it is a fine app for blogs.The issue has not been assigned since Dec 2023 which shows how interested developers are in promoting the use of ProcessWire!!

**Comment by kathep** [comment]
Processwire is useful for so much more than blogs! It's a very versatile and lean system. I can't say enough good things about it. It has been my main content management system for work for a decade. Integrating that system with my other Yunohost setup will be a dream come true!

Since posting earlier I have sought help from the Processwire forum [here](https://processwire.com/talk/topic/26189-reset-admin-password-or-add-superuser-rol/#comment-238643) and the Yunohost forum [here](https://forum.yunohost.org/t/processwire-package-config-how-to-make-nginx-and-apache-play-nicely-together/28802) and [here](https://forum.yunohost.org/t/yunohost-package-404-on-admin-page-after-first-successful-install/27695/2)

Since my last post, I got the admin login to appear... but it won't accept any credentials.... yet.
 
No success yet, but I have found clues. Processwire is made to run on Apache2, Yunohost runs on nginx. So the Processwire .conf files need adjusting to make all rules that are usually in Processwire's .htaccess file work. I have been doing some trial and error experiments. There will definitely be a successful way - @LostKobrakai, a real Processwire wizard, has shared a [gist](https://gist.github.com/LostKobrakai/b895e2e0e8a2c14b4da88cc7e16cf954) for a similar set up.

And the Processwire moderators on the forum have asked for more detail about how Yunohost uses nginx, which is kind and helpful of them. So hopefully, over time, we'll move closer to finding out how to resolve this issue. 

When we know if Yunohost uses nginx as a reverse proxy, then we may get somewhere.
